<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSkinPreview</class>
 <widget class="QDialog" name="DialogSkinPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="windowFrameTopLeft">
     <property name="minimumSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeFDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Resource/frameTopLeft.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QFrame" name="windowFrameTop">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#windowFrameTop {
	border-left: 85;
	padding-left: -85;
	border-image: url(:/Resource/frameTop.png);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayoutTitlebar">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="1" rowspan="2">
         <widget class="QFrame" name="titlebarFrame">
          <property name="styleSheet">
           <string notr="true">QFrame {
	background-color: transparent;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolButton" name="windowIcon">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 0px solid transparent;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Resource.qrc">
               <normaloff>:/Resource/qtlogo-64.png</normaloff>:/Resource/qtlogo-64.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="windowText">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight: bold;</string>
             </property>
             <property name="text">
              <string>QSkinDialog</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3" rowspan="2">
         <widget class="QFrame" name="titlebarButtonsFrame">
          <property name="styleSheet">
           <string notr="true">/*These move the buttons up so they aren't displayed in the center of the titlebar
   Remove these to center your buttons on the titlebar.*/
QFrame#titlebarButtonsFrame {
	padding-top: -1;
	padding-bottom: 10;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayoutTitlebarButtons">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QToolButton" name="minimizeButton">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>18</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>18</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Minimize</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton {
	border: 0px solid transparent;
	border-image: url(:/Resource/minButton.png);
}

QToolButton:hover {
	border-image: url(:/Resource/minButtonH.png);
}

QToolButton:disabled {
	border-image: url(:/Resource/minButtonD.png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QToolButton" name="maximizeButton">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>18</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>18</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Maximize</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton {
	border: 0px solid transparent;
	border-image: url(:/Resource/maxButton.png);
}

QToolButton:hover {
	border-image: url(:/Resource/maxButtonH.png);
}

QToolButton:disabled {
	border-image: url(:/Resource/maxButtonD.png);
}

QToolButton:checked { /*The button will be checked when maximized is set*/
	
	border-image: url(:/Resource/restoreButton.png);
}

QToolButton:checked:hover {
	
	border-image: url(:/Resource/restoreButtonH.png);
}

QToolButton:checked:disabled {
	
	border-image: url(:/Resource/restoreButtonD.png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QToolButton" name="closeButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>44</width>
               <height>18</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>44</width>
               <height>18</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Close</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton {
	border: 0px solid transparent;
	border-image: url(:/Resource/quitButton.png);
}

QToolButton:hover {
	border-image: url(:/Resource/quitButtonH.png);
}

QToolButton:disabled {
	border-image: url(:/Resource/quitButtonD.png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <spacer name="horizontalSpacerTitlebar">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QFrame" name="windowFrameTopRight">
     <property name="minimumSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeBDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Resource/frameTopRight.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QFrame" name="windowFrameLeft">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>12</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>12</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeHorCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Resource/frameLeft.png);
border-left: 1px;
border-top: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QFrame" name="windowFrameRight">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>12</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>12</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeHorCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	border-image: url(:/Resource/frameRight.png);
	border-right: 1;
	border-top: 10;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widgetContents" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="layoutContents">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditAuthor">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Version</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditVersion">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEditDescription">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="windowFrameBottomLeft">
     <property name="minimumSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeBDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Resource/frameBottomLeft.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QFrame" name="windowFrameBottom">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>12</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>12</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeVerCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Resource/frameBottom.png);
border-bottom: 1;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QFrame" name="windowFrameBottomRight">
     <property name="minimumSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeFDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Resource/frameBottomRight.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
